build.xml

```diff
@@ -24,6 +24,7 @@
 	<path id="project.class.path">
 		<fileset dir="lib">
 			<include name="**/*.jar"/>
+			<exclude name="checkstyle-6.19-all.jar"/>
```

Comment:
Why is checkstyle excluded?

Reply:
It contains same FQNs as in some other jar file.
The compiler complains because it uses the classes from this one instead of from the other.

Excluding it when compiling the project is fine because it's not needed there.
It is included in the classpath of checkstyle